WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationDetailsViewController+Extensions.swift

```diff
@@ -210,6 +210,252 @@ extension NotificationDetailsViewController
 
 
 
+/// UITableViewCell Subclass Setup
+///
+extension NotificationDetailsViewController
+{
+    func setupCell(cell: NoteBlockTableViewCell, blockGroup: NotificationBlockGroup) {
+        // Temporarily force margins for WPTableViewCell hack.
+        // Brent C. Jul/19/2016
```

Comment:
Can we remove the name+date comment please? It'll show up on git history already.